<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRUD</class>
 <widget class="QMainWindow" name="CRUD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CRUD</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="titre">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="title">
     <string>GESTION DES PRODUITS :</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>801</width>
       <height>551</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Ajouter un produit</string>
      </attribute>
      <widget class="QLabel" name="reference">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>130</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Réfèrence :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nbv_e">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>220</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="type">
       <property name="geometry">
        <rect>
         <x>194</x>
         <y>220</y>
         <width>163</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Nombre de voyageurs</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="destination_e">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>160</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="reference_e">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>130</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="quantite">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>160</y>
         <width>95</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Destination :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="date_vol_e">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>190</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="date_exp">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>190</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Date de vol:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>260</y>
         <width>131</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>Modifier produit</string>
      </attribute>
      <widget class="QLineEdit" name="reference_e_3">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>140</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>140</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Réfèrence :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>110</y>
         <width>321</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Saisir la réfèrence de produit à modifier:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>360</y>
         <width>201</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Modifier le produit</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="date_vol_e_2">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>200</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="quantite_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>170</y>
         <width>95</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Destination :</string>
       </property>
      </widget>
      <widget class="QLabel" name="type_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>230</y>
         <width>163</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Nombre de voyageurs</string>
       </property>
      </widget>
      <widget class="QLabel" name="date_exp_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>200</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Date de vol:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="destination_e_2">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>170</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nbv_e_2">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>230</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>Supprimer un produit</string>
      </attribute>
      <widget class="QLabel" name="reference_sup">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Réfèrence :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>170</y>
         <width>201</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Supprimer le produit</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="reference_sup_e">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>140</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>110</y>
         <width>321</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Saisir la réfèrence de produit à supprimer:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>Afficher un produit</string>
      </attribute>
      <widget class="QTableView" name="tabproduit_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>10</y>
         <width>591</width>
         <height>451</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>108</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>tous les vols</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>reference</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri selon:</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>destination</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>169</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>nombre de voyageurs</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>92</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ascending</string>
       </property>
      </widget>
      <widget class="QPushButton" name="statbtn">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>300</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Stat</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_12">
      <attribute name="title">
       <string>Chercher et trier les produits</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>391</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Trier les produits</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>150</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Réfèrence :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>190</y>
         <width>201</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Chercher le produit</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_16">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>150</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>120</y>
         <width>321</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Saisir la réfèrence de produit à chercher:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_13">
      <attribute name="title">
       <string>Envoyez des e-mails</string>
      </attribute>
      <widget class="QPushButton" name="exitBtn">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>370</y>
         <width>121</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>641</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <property name="rowWrapPolicy">
         <enum>QFormLayout::WrapLongRows</enum>
        </property>
        <property name="verticalSpacing">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Smtp-server:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Server port:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>Recipant to:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>Subject:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>Message:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="server">
          <property name="text">
           <string>smtp.gmail.com</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="port">
          <property name="text">
           <string>465</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="uname"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="paswd">
          <property name="inputMask">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="rcpt"/>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="subject"/>
        </item>
        <item row="6" column="1">
         <widget class="QPlainTextEdit" name="msg"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="sendBtn">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>370</y>
         <width>131</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Envoyer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_14">
      <attribute name="title">
       <string>Statistiques</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
